cogs5e/initTracker.py

```diff
@@ -934,6 +934,7 @@ async def effect(self, ctx, target_name: str, effect_name: str, *args):
         __General__
         -ac <ac> - modifies ac temporarily; adds if starts with +/- or sets otherwise.
         -sb <save bonus> - Adds a bonus to all saving throws.
+        -cb <check bonus> - Adds a bonus to all attribute checks.
```

Comment:
"ability checks" feels like better wording (see Lucky/Bardic Inspiration for examples of this wording)